python/pyarrow/parquet.py

```diff
@@ -675,7 +675,8 @@ def read_pandas(source, columns=None, nthreads=1, metadata=None):
 
 
 def write_table(table, where, row_group_size=None, version='1.0',
-                use_dictionary=True, compression='snappy', **kwargs):
+                use_dictionary=True, compression='snappy',
+                use_deprecated_int96_timestamps=False, **kwargs):
```

Comment:
We might want to create some kind of `WriteOptions` object soon since we're likely to accumulate plenty more options (like casting timestamps to milliseconds)